### PR TITLE
Add h2-h1c passthrough load-test

### PR DESCRIPTION
**Pull request by ayeshLK**
Related to [#2792](https://github.com/ballerina-platform/ballerina-standard-library/issues/2792)

## Examples
N/A

## Checklist
- [X] Linked to an issue
- [ ] ~Updated the changelog~
- [ ] ~Added tests~
- [ ] ~Updated the spec~

### CODE REVIEW

File: load-tests/h2_h1c_passthrough/src/h2_h1c_passthrough.bal [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        }
+    }
+});
+
+final http:Client nettyEP = check new("http://netty:8688");
+
+service /passthrough on securedEP {
+    resource function post .(http:Caller caller, http:Request clientRequest) {
+        http:Response|http:ClientError response = nettyEP->forward("/service/EchoService", clientRequest);
+        if (response is http:Response) {
```
Comment (chamil321) [inline load-tests/h2_h1c_passthrough/src/h2_h1c_passthrough.bal:35]:
```suggestion
        if response is http:Response {
```

File: load-tests/h2_h1c_passthrough/src/h2_h1c_passthrough.bal [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+            error? result = caller->respond(response);
```
Comment (chamil321) [inline load-tests/h2_h1c_passthrough/src/h2_h1c_passthrough.bal:36]: Shall we return `Response`

File: load-tests/h2_h1c_passthrough/src/h2_h1c_passthrough.bal [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+            path: "./security/ballerinaKeystore.p12",
+            password: "ballerina"
+        }
+    }
+});
+
+final http:Client nettyEP = check new("http://netty:8688");
+
+service /passthrough on securedEP {
+    resource function post .(http:Caller caller, http:Request clientRequest) {
```
Comment (chamil321) [inline load-tests/h2_h1c_passthrough/src/h2_h1c_passthrough.bal:33]: We can avoid caller and return response

File: load-tests/h2_h1c_passthrough/src/h2_h1c_passthrough.bal [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+        http:Response|http:ClientError response = nettyEP->forward("/service/EchoService", clientRequest);
+        if (response is http:Response) {
+            error? result = caller->respond(response);
+        } else {
+            log:printError("Error at h2_h1c_passthrough", 'error = response);
+            http:Response res = new;
+            res.statusCode = 500;
```
Comment (chamil321) [inline load-tests/h2_h1c_passthrough/src/h2_h1c_passthrough.bal:40]: Let's use Status code record here and return

File: load-tests/h2_h1c_passthrough/src/h2_h1c_passthrough.bal (hunk 2 above)
Comment (ayeshLK) [reply load-tests/h2_h1c_passthrough/src/h2_h1c_passthrough.bal]: Fixed with [700054fcb770e253f9bd6bd13dd21b90e4c1c623](https://github.com/ballerina-platform/module-ballerina-http/pull/937/commits/700054fcb770e253f9bd6bd13dd21b90e4c1c623)

File: load-tests/h2_h1c_passthrough/src/h2_h1c_passthrough.bal (hunk 4 above)
Comment (ayeshLK) [reply load-tests/h2_h1c_passthrough/src/h2_h1c_passthrough.bal]: Fixed with [700054fcb770e253f9bd6bd13dd21b90e4c1c623](https://github.com/ballerina-platform/module-ballerina-http/pull/937/commits/700054fcb770e253f9bd6bd13dd21b90e4c1c623)

File: load-tests/h2_h1c_passthrough/src/h2_h1c_passthrough.bal (hunk 3 above)
Comment (ayeshLK) [reply load-tests/h2_h1c_passthrough/src/h2_h1c_passthrough.bal]: Fixed with [5944412578aed9070e9a70ae1889f46c34b28648](https://github.com/ballerina-platform/module-ballerina-http/pull/937/commits/5944412578aed9070e9a70ae1889f46c34b28648)

File: load-tests/h2_h1c_passthrough/scripts/load_test/main.bal [hunk 5]
```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (dilanSachi) [inline load-tests/h2_h1c_passthrough/scripts/load_test/main.bal:1]: Cant we use default jmeter instead of the ballerina script?

Comment (ayeshLK) [reply load-tests/h2_h1c_passthrough/scripts/load_test/main.bal:1]: AFAIK we cannot test HTTP2 services using jmeter.

**Review by dilanSachi — APPROVED** [review]

**Review by chamil321 — APPROVED** [review]

**Pull request merged**